com.moddb.TotalChaos.yaml

```diff
@@ -0,0 +1,80 @@
+app-id: com.moddb.TotalChaos
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '18.08'
+command: gzdoom.sh
+finish-args:
+- --device=dri
+- --socket=wayland
+- --socket=x11
+- --share=ipc
+- --socket=pulseaudio
+- --env=DOOMWADDIR=/app/share/games/doom
+- --persist=.config/gzdoom
+cleanup:
+- /include
+- /lib/*.a
+- /lib/*.la
+- /lib/pkgconfig
+- /share/man
+modules:
+- name: totalchaos
+  buildsystem: simple
+  sources:
+  - type: archive
+    url: https://www.dropbox.com/s/u79pxhjoirdrlji/totalchaos_pk3only_097_4.zip?dl=1
+    sha256: f663fdff884814344b0d8f55f95097a19df4311cff394c0947fdb7ff7b1c32b1
+    dest-filename: totalchaos_pk3only_097_4.zip
+  - type: file
+    path: com.moddb.TotalChaos.desktop
+  - type: file
+    path: com.moddb.TotalChaos.appdata.xml
+  - type: file
+    path: com.moddb.TotalChaos.png
+  build-commands:
+  - install -Dm 644 tc_core*.pk3 -t /app/share/games/doom
+  - install -Dm 644 totalchaos.pk3 -t /app/share/games/doom
+  - install -Dm 644 com.moddb.TotalChaos.appdata.xml -t /app/share/appdata
+  - install -Dm 644 com.moddb.TotalChaos.desktop -t /app/share/applications
+  - install -Dm 644 com.moddb.TotalChaos.png -t /app/share/icons/hicolor/128x128/apps
+
+- name: fluidsynth
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+  - -DLIB_SUFFIX=
+  sources:
+  - type: archive
+    url: https://github.com/FluidSynth/fluidsynth/archive/v2.0.1.tar.gz
+    sha256: 0d80f0b2fb0d3f93722844e654d5b5f004bf31a07f752c0be939b7242ac33248
+
+- name: gzdoom
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+  - -DSEND_ANON_STATS=OFF
+  sources:
+  - type: archive
```

Comment:
Just indent all of this.. https://github.com/flathub/flathub/wiki/YAML-Style-Guide